<launch>

  <node pkg="final_project" type="navigator.py" name="turtlebot_navigator" output="screen" />
  
</launch>
